armory/scenarios/audio_classification.py

```diff
@@ -42,13 +42,16 @@ def _evaluate(self, config: dict) -> dict:
                 f"Fitting model {model_config['module']}.{model_config['name']}..."
             )
             fit_kwargs = model_config["fit_kwargs"]
+            if self.check:
+                fit_kwargs["nb_epochs"] = 1
```

Comment:
So if users submit their own scenarios (which is likely) will be requiring this as part of their _evaluate method? Maybe we should consider the `check` flag modifying the configuration dictionary?